examples/03_model_scoring.py

```diff
@@ -137,18 +137,24 @@
 # can then be pose process using some of the utility functions Earth-2 MIP provides.
 
 # %%
+import numpy as np
 from earth2mip.inference_medium_range import save_scores, time_average_metrics
 
-time = datetime.datetime(2017, 1, 1, 0)
-initial_times = [time + datetime.timedelta(days=30 * i) for i in range(12)]
+# Use 52 initializations.
+time = datetime.datetime(2017, 1, 2, 0)
+initial_times = [time + datetime.timedelta(days=7 * i) for i in range(52)]
+initial_times = [
+    initial_time + datetime.timedelta(hours=6 * int(np.random.choice([0, 1, 2, 3])))
+    for initial_time in initial_times
+]  # randomly select a time between 00, 06, 12 and 18 hrs.
 
 # Output directoy
 output_dir = "outputs/03_model_scoring"
 if not os.path.exists(output_dir):
     os.makedirs(output_dir, exist_ok=True)
     output = save_scores(
         model,
-        n=20,  # 12 hour timesteps
+        n=56,  # 6 hour timesteps (14-day forecast)
```

Comment:
Can we make this just a 7 day forecast?

Reply:
Made it 7 day, and added some commentary on how the n relates to forecast length.